app/domain/output_document.rb

```diff
@@ -3,10 +3,84 @@ class OutputDocument
 
   attr_accessor :appointment_summary
 
+  delegate :id, :guider_name, :income_in_retirement, :continue_working, :unsure,
+           :leave_inheritance, :wants_flexibility, :wants_security,
+           :wants_lump_sum, :poor_health,
+           to: :appointment_summary
+
+  delegate :address_line_1, :address_line_2, :address_line_3, :town, :county,
+           :postcode,
+           to: :appointment_summary, prefix: :attendee
+
   def initialize(appointment_summary)
     @appointment_summary = appointment_summary
   end
 
+  def attendee_name
+    "#{appointment_summary.title} #{appointment_summary.first_name} #{appointment_summary.last_name}".squish
+  end
+
+  def appointment_date
+    appointment_summary.date_of_appointment.to_s(:long)
+  end
+
+  def guider_organisation
+    if appointment_summary.guider_organisation == 'tpas'
+      'The Pensions Advisory Service'
+    else
+      'Pension Wise'
+    end
+  end
+
+  def value_of_pension_pots
+    return 'No value given' if appointment_summary.value_of_pension_pots.blank?
+
+    pension_pot = number_to_currency(appointment_summary.value_of_pension_pots, precision: 0)
+
+    if appointment_summary.upper_value_of_pension_pots.blank?
+      pension_pot
+    else
+      pension_pot + ' to ' + \
+        number_to_currency(appointment_summary.upper_value_of_pension_pots, precision: 0)
+    end
+  end
+
+  def variant
+    if appointment_summary.eligible_for_guidance?
+      if appointment_summary.custom_guidance?
+        'tailored'
+      else
+        'generic'
+      end
+    else
+      'other'
+    end
+  end
+
+  def greeting
+  end
+
+  def appointment_reference
+    appointment_summary.reference_number
```

Comment:
There is a chance we could have multiple summaries for the same appointment. To cover that off I used the following reference in the documents issued to date:

```rb
"#{appointment_summary.id}/#{appointment_summary.reference_number}" # 54/12344
```